flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/catalog/BasicOperatorTable.scala

```diff
@@ -199,6 +198,20 @@ class BasicOperatorTable extends ReflectiveSqlOperatorTable {
     ScalarSqlFunctions.REGEXP_REPLACE,
     SqlStdOperatorTable.TRUNCATE,
 
+    // TIME FUNCTIONS
+    SqlStdOperatorTable.YEAR,
+    SqlStdOperatorTable.QUARTER,
+    SqlStdOperatorTable.MONTH,
+    SqlStdOperatorTable.WEEK,
+    SqlStdOperatorTable.HOUR,
+    SqlStdOperatorTable.MINUTE,
```

Comment:
Do these added operators have their implementations now ?

Reply:
yes, there are already  several IT cases to test them.